spec/support/sdam_monitoring/standalone.yml

```diff
@@ -0,0 +1,74 @@
+description: "Monitoring a standalone connection"
+uri: "mongodb://a:27017"
+phases:
+  -
+    responses:
+      "a:27017":
+        ok: 1
+        ismaster: true
+        minWireVersion: 0
+        maxWireVersion: 4
+    outcome:
+      events:
+        -
+          topology_opening_event:
+            topologyId: "42"
+        -
+          server_opening_event:
+            topologyId: "42"
+            address: "a:27017"
+        -
+          server_description_changed_event:
+            topologyId: "42"
+            address: "a:27017"
+            previousDescription:
+              address: "127.0.0.1:27017"
```

Comment:
The address should be "a:27017" in previousDescription and type "Unknown".  I don't see how it makes sense to have an empty address for a server description changed event.  The address is part of the top-level event metadata, so both descriptions should share it.